Makefile.am

```diff
@@ -291,9 +302,11 @@ libini_config_la_LIBADD = \
     libpath_utils.la \
     libref_array.la \
     libbasicobjects.la
-libini_config_la_LDFLAGS = \
-    -version-info 7:1:2 \
-    -Wl,--version-script=$(top_srcdir)/ini/libini_config.sym
+libini_config_la_LDFLAGS = -liconv -lintl \
```

Comment:
These libs should not be part of `*_LDFLAGS` but `*_LIBADD` but that is not important ATM.

The main issue is that they are not detected at configure time and they are hardcoded and therefore linking fails on platforms which does not have such libraries.

```
  CCLD     libini_config.la
/usr/bin/ld: cannot find -liconv
/usr/bin/ld: cannot find -lintl
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:1203: libini_config.la] Error 1
make[1]: Leaving directory '/home/alcik/projects/ding-libs'
make: *** [Makefile:1015: all] Error 2
```

But that should be solved as a separate commit. BTW I noticed that on FreeBSD as well but did not have a time to find a portable solution.

Reply:
The changes in a267b038d6c15fc1b7e42c0c08b1d7c6488c8a4a to make autoconf detect the need for those, plus "ACLOCAL_PATH=/usr/local/Cellar/gettext/0.19.8.1/share/aclocal/ autoreconf -i" on macOS (but which should hopefully just work on FreeBSD with gettext installed) should fix this